### PR TITLE
update contracts

### PR DESCRIPTION
**Pull request by MixailE**
Changed the `checkAttestationHash` function to use the new contact methods `attestationHashesEntangled`, `maxEntanglementsPerAttestationType`

See contract changes here [multiple-attestation-uses-v2](https://github.com/BigWhaleLabs/ketl-attestation-token/tree/multiple-attestation-uses-v2)

### CODE REVIEW

**Comment by MixailE** [comment]
[Identify breaking functions in ketl-app and ketl-zk-frontend and ship changes.](https://trello.com/c/gvjMtOrK/3372-identify-breaking-functions-in-ketl-app-and-ketl-zk-frontend-and-ship-changes)

File: src/helpers/checkAttestationHash.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
 import ketlAttestationContract from 'helpers/ketlAttestationContract'
 
-export default function checkAttestationHash(hash: string) {
-  return ketlAttestationContract.attestationHashesEntangled(hash)
+export default async function checkAttestationHash(
+  hash: string,
+  attestationType: number
```
Comment (T-Damer) [inline src/helpers/checkAttestationHash.ts:6]: Should be `enum`

Comment (wilsonplau) [reply src/helpers/checkAttestationHash.ts:6]: Agreed, but would prefer to do this in a separate PR.

**Review by wilsonplau — APPROVED** [review]
👍🏼

**Pull request merged**